Report/sections/2_design/Design-Choice

```diff
@@ -0,0 +1,10 @@
+
+\section{Design choice}
+
+
+The language is intended to be a transitional language from visual programming to high-level language. Based on the problem analysis, it has been possible to specify the design for the user so that it hits the target group.
+ The target group is beginners in programming who are younger people. Therefore a programming language to make computer games has been built. This has been chosen as young people can relate more to video games as their hobby. 
```

Comment:
Kunne godt uddybes en del:
"This has been chosen as young people can relate more to video games as their hobby" -  skrev vi ikke lidt om det her i starten? måske lidt at hente derfra. 
Men i hvert fald en kilde på at understøtte påstanden.